src/Ole32/Ole32.Helpers.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Methods and nested types that are not strictly P/Invokes but provide
+    /// a slightly higher level of functionality to ease calling into native code.
+    /// </content>
+    public static partial class Ole32
+    {
+        // This is where you define methods that assist in calling P/Invoke methods.
+        // For example, if a P/Invoke method requires allocating unmanaged memory
+        // and freeing it up after the call, a helper method in this file would
+        // make "P/Invoking" for most callers much easier and is a welcome addition.
+
+        /// <summary>
+        /// Enables cancellation of synchronous calls on the calling thread.
+        /// </summary>
+        /// <returns>This function can return the standard return values <see cref="HResult.Code.S_OK"/>, <see cref="HResult.Code.E_FAIL"/>, <see cref="HResult.Code.E_INVALIDARG"/>, and <see cref="HResult.Code.E_OUTOFMEMORY"/>.</returns>
+        public static HResult CoEnableCallCancellation()
+        {
+            return CoEnableCallCancellation(IntPtr.Zero);
```

Comment:
We don't typically offer helper methods that simply remove a parameter. We generally reserve them for assisting with difficult memory management. 
What if we remove these two helper methods and add `= default(IntPtr)` to the `[dllimport]` methods themselves? It should accomplish the same thing.

Reply:
Thanks. Did you want to use the `= null` default parameter approach?